docs/index.md

```diff
@@ -0,0 +1 @@
+# code-client
```

Comment:
Let's write at least a sentence what the library is about. You can copy it from the existing readme.